include/openssl/bio.h

```diff
@@ -724,10 +724,16 @@ void BIO_copy_next_retry(BIO *b);
  * long BIO_ghbn_ctrl(int cmd,int iarg,char *parg);
  */
 
-# ifdef __GNUC__
-#  define __bio_h__attr__ __attribute__
-# else
-#  define __bio_h__attr__(x)
+# define __bio_h__attr__(x)
+# if defined(__GNUC__) && defined(__STDC_VERSION__)
+    /*
+     * Because we support the 'z' modifier, which made its appearance in C99,
+     * we can't use __attribute__ with pre C99 compilers.
```

Comment:
> Regarding information

Just to be clear. I was not questioning reference to specifically C99 *standard*, but merely formulating it as "pre-C99 *compiler*". Because "Cxx compiler" kind of implies one that is explicitly compliant with single standard. Or rather up to xx. But if you look closer, you wouldn't help noticing that it's rather question of what is the default. Most notably gcc-4.8 (one used by Travis) is capable of assuming c9x and even c11 identities, it just doesn't by default, i.e. without you passing explicit -std option... Thanks for listening :-)

As for &&. As implied, it's occupational hazard of non-native speaker. It's not like I don't understand what it says/implies, I'm just not sure that there is no ambiguity in wording, that's all.